spec/controllers/api/v1/email_preferences_controller_spec.rb

```diff
@@ -1,5 +1,54 @@
 require 'rails_helper'
 
 RSpec.describe Api::V1::EmailPreferencesController, type: :controller do
+  login_user
 
+  describe "PUT #update" do
+    context "when user has no email preference" do
+      it "should return success" do
+        put :update, format: :json, params: {email_preference: {news: true}}
+
+        assert_response :success
+      end
+
+      it "should create and return new email preference" do
+        email_preference = {"news" => true, "likes" => false, "messages" => false}
```

Comment:
Could you create an email factory instead, as it's an actual model.
https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#using-factories

you can find other factories in this folder spec/factories